apps/web/components/GardenCard.tsx

```diff
@@ -1,62 +1,65 @@
+import { Card } from "@/components/Card";
+import { getChain } from "@/configs/chainServer";
 
 type TokenGarden = getTokenGardensQuery["tokenGardens"][number];
 
 export function GardenCard({ garden }: { garden: TokenGarden }) {
-  const { id, name, symbol, chainId } = garden;
+  const { id, name, chainId } = garden;
   const link = `/gardens/${chainId}/${id}`;
   const communities = garden.communities?.filter((comm) => comm.isValid);
   const commLength = communities?.length ?? 0;
-  const totalMembers =
-    communities
-      ?.map((comm) => comm.members?.length ?? 0)
-      .reduce((a, b) => a + b, 0) ?? 0; //@todo temporary, that can be take from the subgraph
+
+  // TODO: improve the types ?
+  function countUniqueMemberAddresses(communitiesArray: any[] | undefined) {
+    const uniqueAddresses = new Set();
+
+    communitiesArray?.forEach(community => {
+      if (community.members && Array.isArray(community.members)) {
+        community.members.forEach((member: { memberAddress: Address; }) => {
+          if (member.memberAddress) {
+            uniqueAddresses.add(member.memberAddress);
+          }
+        });
+      }
+    });
+    return uniqueAddresses.size;
+  }
+
+  const totalMembers = countUniqueMemberAddresses(communities);
+
   return (
-    <div className="border2 bg-surface relative flex max-w-[320px] flex-col overflow-hidden rounded-lg border-black shadow">
-      <div className="flex flex-col gap-2 p-2">
-        <div className="card relative" />
-        <div className="flex flex-col items-center gap-1">
-          <h4 className="text-2xl font-bold">{name}</h4>
-          <p className="text-center">{symbol}</p>
-          <div className="align-center flex justify-center">
-            <p className="text-center text-xs">Network:</p>
-            <div className="mx-1 content-center justify-center">
-              <ChainIcon chain={chainId} height={16} />
+    <Card href={link} >
+      <div className="flex flex-col gap-7">
+        <div className="flex items-center justify-center gap-2">
+          {/* TODO: find appropiate token image */}
+          <TokenLabel chainId={chainId} noSymbol />
+          <h3 className="text-neutral-content">{name}</h3>
+        </div>
+        <div className="flex flex-col gap-4">
+          <div className="flex flex-col">
+            <div className="align-start flex flex-col justify-start">
+              <p className="text-neutral-content text-sm">Network:</p>
+              <div className="flex gap-2.5 items-center">
+                <h5 className="text-neutral-content">{getChain(chainId)?.name}</h5>
+              </div>
             </div>
-            <p className="text-center text-xs">{getChain(chainId)?.name}</p>
+          </div>
+          <div className="flex flex-col gap-1">
+            <Statistic label={`Communit${commLength > 1 ? "ies" : "y"}`} count={commLength} icon={<BuildingStorefrontIcon />} />
+            <Statistic label={"Members"} count={totalMembers} />
```

Comment:
i would suggest to have icons for all of them or for none

Reply:
it was a merge problem, it is both with icons, thankss lucho